protobuf/transform.go

```diff
@@ -189,32 +189,35 @@ func defaultOptionsForScannedMessage(s *scanner.Struct) Options {
 	}
 }
 
-func (t *Transformer) transformField(pkg *Package, field *scanner.Field, pos int) *Field {
+func (t *Transformer) transformField(pkg *Package, msg *Message, field *scanner.Field, pos int) *Field {
 	var (
 		typ      Type
 		repeated = field.Type.IsRepeated()
 	)
 
+	pField := &Field{
```

Comment:
what is `pField`? if it's because of the conflicting `field` var i'd rather call it `f`